events/app_watcher.go

```diff
@@ -222,6 +254,15 @@ func (m *AppWatcher) processLogMessage(logMessage *sonde_events.LogMessage) erro
 	return nil
 }
 
+func (m *AppWatcher) processHttpStartStopMetric(httpStartStop *sonde_events.HttpStartStop) {
+	responseDuration := float64(time.Duration(httpStartStop.GetStopTimestamp() - httpStartStop.GetStartTimestamp())) / float64(time.Second)
```

Comment:
```suggestion
	responseDuration := time.Duration(httpStartStop.GetStopTimestamp() - httpStartStop.GetStartTimestamp()).Seconds()
```
time.Duration has a [Seconds() method](https://golang.org/pkg/time/#Duration.Seconds) which is probably more intention-revealing than this expression

Reply:
we could also store the variable as type `time.Duration` and only convert to `Seconds()` when we pass it to `Observe()` below. choices, choices..